front/components/actions/tables_query/TablesQueryActionDetails.tsx

```diff
@@ -92,78 +93,64 @@ function QueryThinking({ action }: { action: TablesQueryActionType }) {
   );
 }
 
-type TablesQueryActionWithResultsType = TablesQueryActionType & {
-  output: {
-    query_title?: string;
-    results: unknown[];
-  };
-};
-
-function hasQueryResults(
-  action: TablesQueryActionType
-): action is TablesQueryActionWithResultsType {
-  const { output } = action;
-
-  return (
-    output !== null &&
-    typeof output === "object" &&
-    "results" in output &&
-    Array.isArray(output.results)
-  );
-}
-
-function QueryTablesResults({ action }: { action: TablesQueryActionType }) {
+function QueryTablesResults({
+  action,
+  owner,
+}: {
+  action: TablesQueryActionType;
+  owner: WorkspaceType;
```

Comment:
`nit`
Let's use `LightWorkspaceType` when we can:
```suggestion
  owner: LightWorkspaceType;
```